.travis.yml

```diff
@@ -7,16 +7,15 @@ python:
   - 2.7
   - 3.6
   - 3.7-dev
+
 services:
   - docker
```

Comment:
How does Travis start Docker, if Docker installation has been moved to Tox? (below)

Reply:
tox installs only the docker python bindings, not the service itself.